.github/pull_request_template.md

```diff
@@ -0,0 +1,37 @@
+## Summary
+<!--
+    Provide a link to the ticket or document which prompted this change,
+    Describe the rationale and design decisions.
+-->
+
+## Feature Flag
+<!-- If this is specific to a feature flag, which one? -->
+
+## Product Description
+<!-- For non-invisible changes, describe user-facing effects. -->
```

Comment:
I think it could be a good idea to include screenshots or videos for visible changes. So perhaps change the title to `Product Description/Screenshots/Videos`?

Reply:
Sure, sounds good.